### ISSUE TITLE
Missing npm scripts & deps

### ISSUE THREAD

**Issue by alexdevero** [MrChuckomo/poddycast#42]
The app is missing npm scripts and dependencies in `package.json`.  Without them it is hard to contribute to the codebase of the project.

**Comment by alexdevero** [comment]
This fix will allow to run the app locally:
```
{
    "main": "main.js",
    "name": "Poddycast",
    "version": "0.6.1",
    "scripts": {
        "start": "electron ."
    },
    "dependencies": {
        "electron": "^3.0.2"
    }
}
```

**Comment by alexdevero** [comment]
@MrChuckomo you can close this issue. It was solved by PR #43.
